src/chi/components/tables/tables.scss

```diff
@@ -2,8 +2,233 @@
 @import 'variables';
 
 .chi {
-  section {
-    &.a-table {
+
+  th {
+    text-align: left;
+  }
+
+  .a-table {
+    padding-top: 3rem;
+    position: relative;
+    border-spacing: 0;
+    font-size: $text;
+    line-height: $line-height-smaller;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+
+    table {
+      border-spacing: 0;
+      font-size: $text;
+      line-height: $line-height-smaller;
+      margin: 0;
+      padding: 0;
+      width: 100%;
+    }
+
+    tr {
+      background-color: set-color(white);
+      height: 3rem;
+    }
+
+    th {
+      border-bottom: 0.0625rem solid set-color(grey, 30);
+      font-weight: 600;
+      height: 0;
+      padding: 1rem;
+      text-transform: capitalize;
+
+      &.-sorted {
+        div {
+          align-items: center;
+          cursor: pointer;
+          display: flex;
+          flex-direction: row;
+
+          &::after {
+            background-image: url('./icon-chevron-thin-up.svg');
+            content: '';
+            display: block;
+            height: 0.75rem;
+            margin-left: 0.5rem;
+            opacity: 0;
+            width: 0.75rem;
+          }
+        }
+
+        & i {
+          margin-left: 0.5rem
+        }
+      }
+
+      &.-ascending {
+        div::after {
+          opacity: 1;
+        }
+      }
+
+      &.-descending {
+        div::after {
+          opacity: 1;
+          transform: rotate(180deg);
+        }
+      }
+    }
+
+    & td {
+      border-bottom: 0.0625rem solid set-color(grey, 20);
+      padding: 1rem;
+      padding-bottom: 0.9375rem;
+      vertical-align: middle;
+    }
+
+    & tbody {
+      tr {
+        &.-row--success {
+          background-color: set-color(green, 10);
+        }
+
+        &.-row--warning {
+          background-color: set-color(yellow, 10);
+        }
+
+        &.-row--danger {
+          background-color: set-color(red, 10);
+        }
+
+        &.-row--info {
+          background-color: set-color(blue, 10);
+        }
+
+        &[disabled] {
+          color: rgba(set-color(grey, 90), 0.4);
+        }
+      }
+    }
+
+    &.-hover {
+      tbody {
+        tr {
+          &:hover,
+          &.-hover {
+            background-color: set-color(grey, 10);
+          }
+        }
+      }
+    }
+
+    &.-striped {
+      tbody {
+        tr:nth-child(odd) {
+          background-color: set-color(grey, 10);
+        }
+      }
+    }
+
+    &.-borderless {
+      &::before {
+        border: 0;
+      }
+
+      th {
+        border-bottom: 0.0625rem solid set-color(grey, 30);
```

Comment:
`-borderless` modifier is still rendering a border on the `th`. need to set to 0